awscli/compat.py

```diff
@@ -57,6 +57,12 @@ def __exit__(self, type, value, traceback):
             import msvcrt
             msvcrt.setmode(sys.stdout.fileno(), self.previous_mode)
 
+def ensure_text_type(s, encoding='utf-8', errors='strict'):
```

Comment:
You don't use or test `encoding` or `errors`. It's probably best to get rid of them until they're needed.